.github/pull_request_template.md

```diff
@@ -1,6 +1,10 @@
-Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+📌 Review Process TL;DR:
```

Comment:
I'm still not sure that this should be in the PR template, the CONTRIBUTING should be the place where everything is defined, and we expect people to read it. The goal of the template is to define a skeleton on which contributors can start. If the team agree to have this, it's fine for me, but keep in mind that we, as maintainer, will have to remove those 5 lines everytimes.